runtime/manta/src/lib.rs

```diff
@@ -631,28 +487,23 @@ construct_runtime!(
         } = 1,
         Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
         ParachainInfo: parachain_info::{Pallet, Storage, Config} = 3,
-        TransactionPause: pallet_tx_pause::{Pallet, Call, Storage, Event<T>} = 9,
+
         // Monetary stuff.
         Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
         TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>} = 11,
 
-        // Collator support. the order of these 5 are important and shall not change.
+        ParachainStaking: pallet_parachain_staking::{Pallet, Call, Storage, Event<T>, Config<T>} = 48,
+        // Collator support.
+        AuthorInherent: pallet_author_inherent::{Pallet, Call, Storage, Inherent} = 60,
+        AuraAuthorFilter: pallet_aura_style_filter::{Pallet, Storage} = 63,
+        // The order of the next 4 is important and shall not change.
         Authorship: pallet_authorship::{Pallet, Call, Storage} = 20,
-        CollatorSelection: pallet_collator_selection::{Pallet, Call, Storage, Event<T>, Config<T>} = 21,
+        CollatorSelection: manta_collator_selection::{Pallet, Call, Storage, Event<T>, Config<T>} = 21,
         Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 22,
         Aura: pallet_aura::{Pallet, Storage, Config<T>} = 23,
-        AuraExt: cumulus_pallet_aura_ext::{Pallet, Storage, Config} = 24,
 
         // Preimage registry.
         Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 28,
-        // System scheduler
-        Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 29,
-
-        // XCM helpers.
-        XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 30,
-        PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 31,
-        CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 32,
-        DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 33,
 
         // Handy utilities.
         Utility: pallet_utility::{Pallet, Call, Event} = 40,
```

Comment:
I don't think Utility and Multisig are actually needed.

Reply:
Btw if we're using multisig for the sudo, we need to make sure that all of the related accounts (and the multisig itself) are funded in the genesis.

Reply:
Thanks for pointing this out, i would've only cared about the controller accounts otherwise.

> Multisig for the sudo i guess, but what is Utility needed for ?
Multisig sudo is annoying to operate, with batch calls we can bundle operations and sign them once